test/gurka/test_build_admin.cc

```diff
@@ -0,0 +1,128 @@
+#include <gtest/gtest.h>
+
+#include "gurka.h"
+
+#include "baldr/admin.h"
+#include "mjolnir/admin.h"
+#include "mjolnir/adminbuilder.h"
+#include "mjolnir/pbfgraphparser.h"
+
+using namespace valhalla;
+using namespace valhalla::baldr;
+using namespace valhalla::mjolnir;
+
+class AdminTest : public ::testing::Test {
+protected:
+  static gurka::map admin_map;
+
+  static void SetUpTestSuite() {
+    constexpr double gridsize = 10;
+
+    const std::string ascii_map = R"(
+          A-------B-------C
+          |       |       |
+          |       |       |
+          |       |       |
+          F-------E-------D
+    )";
+
+    // To define an administrative boundary, the nodes must form a closed polygon.
+    const gurka::ways ways = {{"ABCDEFA",
+                               {
+                                   {"highway", "primary"},
```

Comment:
i dont think the ways need tags at all

Reply:
Lol, yea, I wasn't sure.  It did seem odd to define an admin boundary as a highway. :). I'll look into it.

Reply:
I tried removing the tags and I get a seg fault.  For reasons I haven't figured out yet, despite having a debug build, CLion doesn't break on the seg fault, it just runs/reports it.  Frustrating.

I tried an empty map, an empty value in the map, and a nonsense value in the map - all crash.  But the test is happy with "highway", "primary".

Thoughts?

Reply:
> I tried removing the tags and I get a seg fault. For reasons I haven't figured out yet, despite having a debug build, CLion doesn't break on the seg fault, it just runs/reports it. Frustrating.

@ktatterso are you able to get a backtrace (from coredump maybe)?

Reply:
We discussed and realized the mock map needed one route in it, so I added a highway that spans the two states.